migrations/s3-migrations/migrateToUniqueBuckets.sh

```diff
@@ -1,3 +1,14 @@
-aws s3 mb s3://[new-bucket]
-aws s3 sync s3://[old-bucket] s3://[new-bucket]
-aws s3 rb --force s3://[old-bucket]
\ No newline at end of file
+
+old_buckets=("plots-tables" "cell-sets" "worker-results" "processed-matrix" "biomage-pipeline-debug" "biomage-source" "biomage-filtered-cells" "biomage-backups" "biomage-originals" "biomage-public-datasets") 
+environments=("staging" "production")
+accound_id="242905224710"
```

Comment:
Insted of hardcoding, can you get the account id from AWS profile? (see https://stackoverflow.com/a/40121084/1940886)

Reply:
nice